url_shortener/logic.py

```diff
@@ -12,15 +12,91 @@ def __init__(self, storage: Storage):
         """ Creating an instance of our logic. Pay attention that storage is not created within the logic, but
         passed as a parameter from outside. This is called Dependency Injection.
         """
+        print("Logic is initilized.")
```

Comment:
If these `print` statements were added here for debugging purposes, they should be removed. If they are needed for infromational purposes on application startup, please use `logging` module:

```python
import logging

# The following line will instantiate a logger instance which knows about current module name
# so you will be able to know which module logged which lines
log = logging.getLogger(__name__)

# Simple log. Available functions are `debug`, `info`, `warn`, `error`.
log.info('Your message here')

# You can interpolate some variables into your log message:
answer = 42
log.info('Answer is %s. Unfortunately I do not know the question', answer)
```